package.json

```diff
@@ -64,7 +64,8 @@
     "pg": "^7.12.1",
     "pg-hstore": "^2.3.3",
     "uuid": "^3.3.3",
-    "validate": "^5.1.0"
+    "validate": "^5.1.0",
+    "supertest": "^4.0.2"
```

Comment:
supertest should be in devDependencies

Reply:
Vscode throws an error when it's in devDependencies

Reply:
please check if it is an issue/setting in your vscode or is if it is a linting rule we need to update but it should be in devDependencies.

Reply:
I have fixed this